caikit/core/data_model/base.py

```diff
@@ -532,28 +546,30 @@ def get_proto_class(cls) -> Type[ProtoMessageType]:
         return cls._proto_class
 
     @classmethod
-    def get_field_message_type(cls, field_name: str) -> Optional[Type["DataBase"]]:
-        """Get the data model class for the given field if the field is a
-        message or a repeated message
+    def get_field_message_type(cls, field_name: str) -> Optional[type]:
+        """Get the data model class for the given field. This function relies on
```

Comment:
I think this is slightly more accurate since fields can be primitives too?

```suggestion
        """Get the python type for the given field. This function relies on
```